src/GeoIp2/WebService/Client.php

```diff
@@ -180,7 +188,14 @@ private function responseFor($endpoint, $class, $ipAddress)
 
         $client = $this->guzzleClient ?
             $this->guzzleClient : new GuzzleClient();
-        $request = $client->get($uri, array('Accept' => 'application/json'));
+        $options = array();
+        if($this->timeout != null){
```

Comment:
Yeah, I agree that they are generally the same, but I want it to be clear that we have the same behavior as Guzzle when it is set to `0`.